matlabCode/createBIDS_anat_T1w_json.m

```diff
@@ -0,0 +1,49 @@
+%% Template Matlab script to create an BIDS compatible sub-01_ses-01_acq-highres_run-01_T1w.json file
+% This example lists all required and optional fields.
+% When adding additional metadata please use camelcase
+% 
+%
+% anushkab, 2018
+%%
+clear all
+root_dir = '../';
+project_label = 'templates';
+sub_id = '01';
+ses_id = '01';
+acq_id = 'highres'
+
+acquisition = 'anat';
+run_id='01';
+
+
+anat_json_name = fullfile(root_dir,project_label,...
+  ['sub-' sub_id],...
+              ['ses-' ses_id],acquisition,...
+              ['sub-' sub_id ...
+              '_ses-' ses_id ...
+              '_acq-' acq_id ...
+              '_run-' run_id '_T1w.json']);
+
+% Assign the fields in the Matlab structure that can be saved as a json.
+%The following fields must be defined:
+
+%%
+%The “effective” sampling interval, specified in seconds, between lines in the phase-encoding direction, defined based on the size of the reconstructed image in the phase direction.  It is frequently, but incorrectly, referred to as  “dwell time” (see DwellTime parameter below for actual dwell time).  It is  required for unwarping distortions using field maps. Note that beyond just in-plane acceleration, a variety of other manipulations to the phase encoding need to be accounted for properly, including partial fourier, phase oversampling, phase resolution, phase field-of-view and interpolation. This parameter is REQUIRED if corresponding fieldmap data is present. 
+
+anat_json.EffectiveEchoSpacing='0.0000074';
```

Comment:
There are many missing fields. There are two types of templates we are hoping to provide, one with all required/recommended/optional fields and one with only required fields. There seem to be many more fields than just these two.